releases/release-5.3.0.md

```diff
@@ -348,6 +348,8 @@ Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters
     - Fix the issue of `data too long` error when dumping statistics from the table with `new collation` data [#27024](https://github.com/pingcap/tidb/issues/27024)
     - Fix the issue that the retried transactions' statements are not included in `TIDB_TRX` [#28670](https://github.com/pingcap/tidb/pull/28670)
     - Fix the wrong default value of the `plugin_dir` configuration [#28084](https://github.com/pingcap/tidb/issues/28084)
+    - Fix the issue that the `CONVERT_TZ` function returns `NULL` when it is given a named timezone and a UTC offset [#8311](https://github.com/pingcap/tidb/issues/8311)
+    - Fix the issue that `CREATE SCHEMA` does not use the character set specified by `character_set_server` and `collation_server` for new schemas if none are provided as part of the statement [27216](https://github.com/pingcap/tidb/pull/27216)
```

Comment:
[27216](https://github.com/pingcap/tidb/pull/27216) > [#27214](https://github.com/pingcap/tidb/issues/27214)

Reply:
@shichun-0415 you're saying the link is wrong? I think that needs a new PR if that's the case as this one has been merged already.

Reply:
Each release note should have an issue appended instead of a PR. Also, the numbers are prefixed with `#`.